configs/smartling-config.json

```diff
@@ -0,0 +1,103 @@
+{
+    "locales": [{
+            "smartling": "ar",
+            "application": "ar"
+        },
+        {
+            "smartling": "zh-CN",
+            "application": "zh-CN"
+        },
+        {
+            "smartling": "zh-TW",
+            "application": "zh-TW"
+        },
+        {
+            "smartling": "nl-NL",
+            "application": "nl"
+        },
+        {
+            "smartling": "fr-CA",
+            "application": "fr-CA"
+        },
+        {
+            "smartling": "fr-FR",
+            "application": "fr"
+        },
+        {
+            "smartling": "de-DE",
+            "application": "de"
+        },
+        {
+            "smartling": "hi-IN",
+            "application": "hi-IN"
+        },
+        {
+            "smartling": "id-ID",
+            "application": "id"
+        },
+        {
+            "smartling": "it-IT",
+            "application": "it"
+        },
+        {
+            "smartling": "ja-JP",
+            "application": "ja"
+        },
+        {
+            "smartling": "ko-KR",
+            "application": "ko"
+        },
+        {
+            "smartling": "ms-MS",
+            "application": "ms"
+        },
+        {
+            "smartling": "pl-PL",
+            "application": "pl"
+        },
+        {
+            "smartling": "pt-BR",
+            "application": "pt-BR"
+        },
+        {
+            "smartling": "pt-PT",
+            "application": "pt-PT"
+        },
+        {
+            "smartling": "ru-RU",
+            "application": "ru"
+        },
+        {
+            "smartling": "es",
+            "application": "es"
+        },
+        {
+            "smartling": "es-MX",
+            "application": "es-MX"
+        },
+        {
+            "smartling": "es-ES",
+            "application": "es-ES"
+        },
+        {
+            "smartling": "sw",
+            "application": "sw"
+        },
+        {
+            "smartling": "tr-TR",
+            "application": "tr"
+        },
+        {
+            "smartling": "vi-VN",
+            "application": "vi"
+        }
+    ],
+    "resourceSets": [{
+        "type": "FLUENT",
+        "authorizeContent": "false",
+        "pathRegex": "www-l10n\\/en\\/(?<path>.*)\\.ftl",
```

Comment:
Who in our team is familiar with this configuration file? Where can I find a reference?

Reply:
https://github.com/mozilla-l10n/tooling_test_repo/blob/main/www-l10n-test/configs/smartling-config.json - this is what we tested with. I modified the file to include only Fluent.

Reply:
That's not really a reference to understand the available parameters though. I assume that should be on Smartling somewhere?

Reply:
(found it https://help.smartling.com/hc/en-us/articles/360008154073-Repository-Connector-Translation-Settings)